site/_quarto.yml

```diff
@@ -12,12 +12,12 @@ metadata-files:
 website:
   announcement: 
     content: "[**{{< fa book-open-reader >}} EU AI Act Compliance**](https://validmind.com/download-whitepaper-the-eu-ai-act/) — Read our original regulation brief on how the EU AI Act aims to balance innovation with safety and accountability, setting standards for responsible AI use" 
-    position: below-navbar 
+    position: above-navbar 
   favicon: validmind.png
   title: "ValidMind"
-  cookie-consent:
-    type: express
-    style: simple
+  # cookie-consent:
```

Comment:
Why are you removing the cookie consent here? It's required by law in some regions.